Makefile

```diff
@@ -0,0 +1,17 @@
+BINFILE=build/_output/bin/dedicated-admin-operator
+MAINPACKAGE=./cmd/manager
+GOENV=GOOS=linux GOARCH=amd64 CGO_ENABLED=0
+GOFLAGS=-gcflags="all=-trimpath=${GOPATH}" -asmflags="all=-trimpath=${GOPATH}"
+
+.PHONY: check
+check: ## Lint code
+	gofmt -s -l $(shell go list -f '{{ .Dir }}' ./... ) | grep ".*\.go"; if [ "$$?" = "0" ]; then gofmt -s -d $(shell go list -f '{{ .Dir }}' ./... ); exit 1; fi
+	go vet ./cmd/... ./pkg/...
+
+
+.PHONY: build
+build: ## Build binary
+	${GOENV} go build ${GOFLAGS} -o ${BINFILE} ${MAINPACKAGE}
+
+
+	
```

Comment:
A couple of unnecessary blank lines? Other than that LGTM!

Reply:
I trimmed the extra /n

thanks!